### PR TITLE
Identify solution hotfix

### PR DESCRIPTION
**Pull request by rcooke-ast**
This allows a user to load a master wavelength calibration frame from file. This fixes the old format (json) to the new format (fits)

### CODE REVIEW

File: pypeit/core/wavecal/waveio.py [hunk 1]
```diff
@@ -279,7 +279,7 @@ def load_line_lists(lamps, unknown=False, skip=False, all=False, NIST=False,
                 msgs.warn("Input line {:s} is not included in arclines".format(lamp))
                 msgs.info("Please choose from the following list:" + msgs.newline() +
                           ",".join(all_list))
-                import pdb; pdb.set_trace()
+                embed()
```
Comment (profxj) [inline pypeit/core/wavecal/waveio.py]: maybe remove?  or do we really want to stop here?

**Review by profxj — APPROVED** [review]

File: pypeit/core/wavecal/waveio.py (hunk 1 above)
Comment (rcooke-ast) [reply pypeit/core/wavecal/waveio.py]: Removing is fine with me... I've also changed this to throw an error message.

**Review by kbwestfall — APPROVED** [review]
Looks good to me.  Do we need/have any unit tests for this?

**Comment by rcooke-ast** [comment]
> Looks good to me. Do we need/have any unit tests for this?

Yeah, I don't think we have any tests for this script... I'll add some in during a future PR  :-)

**Pull request merged**